src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,22 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int NEED_YEAR = 35;
```

Comment:
```suggestion
    private static final int AVAILABLE_YEAR = 35;
```
NEED not really  appropriate use in this case